nginx_stage/lib/nginx_stage/user.rb

```diff
@@ -36,6 +36,18 @@ def initialize(user)
       @passwd = Etc.getpwnam user.to_s
       @group = Etc.getgrgid gid
       @groups = get_groups
+
+      if name.to_s != user.to_s
+        err_msg = <<~HEREDOC
+          Username '#{user}' is being mapped to '#{name}' in SSSD and they don't match.
+          Users with domain names cannot be mapped correctly. If '#{name}' still has the
+          domain in it you'll need to set SSSD's full_name_format.
```

Comment:
Do we want to say to set this:

```
full_name_format = %1$s
```

Rather than just mentioning the setting?  Based on man page for `sssd.conf`, that would get you just the username.

Reply:
Done